scripts/update_teams.py

```diff
@@ -53,7 +53,14 @@ def create_team(org, name, description, repos):
         org.url + "/teams",
         input=post_parameters
     )
-    return github.Team.Team(org._requester, headers, data, completed=True)
+    team = github.Team.Team(org._requester, headers, data, completed=True)
+
+    for repo in repos:
+        # Ensure that we have merge rights on the repo for this team.
+        url = team.url + "/repos/" + repo._identity
```

Comment:
We should probably change `repo._identity` to `repo.full_name`. They are the same FWICT, but the latter is part of the PyGithub API.